### PR TITLE
Added and example of instant VU's

### LINKED ISSUE #516: Document instant jumps with 0-duration stages for `ramping-vus` and `ramping-arrival-rate`
Originally discussed in https://github.com/grafana/k6/issues/2252 and https://github.com/grafana/k6/issues/2256, it turns out that we haven't documented how users can have instant jumps in the number of VUs (for [`raming-vus`](https://k6.io/docs/using-k6/scenarios/executors/ramping-vus/) a.k.a. [`options.stages`](https://k6.io/docs/using-k6/options/#stages)) or arrival-rate (for [`ramping-arrival-rate`](https://k6.io/docs/using-k6/scenarios/executors/ramping-arrival-rate/)) by inserting stages with a duration of `0` in their configs, like this:
```js
export const options = {
  scenarios: {
    contacts: {
      executor: 'ramping-arrival-rate',
      preAllocatedVUs: 50,
      startRate: 50,
      stages: [
        { target: 200, duration: '30s' }, // linearly go from 50 iters/s to 200 iters/s
        { target: 500, duration: '0' }, // instantly jump to 500 iters/s
        { target: 500, duration: '10m' }, // continue with 500 iters/s for 10 minutes
      ],
    },
  },
};
```

So we should explain this and add an example.

### PR DESCRIPTION
**Pull request by LaserPhaser**
Closes #516

### CODE REVIEW

**Review by na-- — CHANGES_REQUESTED** [review]
Thanks for making this! There is a slight problem though, the page title and URL are about VU jumps, but the example is about a `ramping-arrival-rate` executor, not about `ramping-vus`. So, in the example, the `rate` instantly jumps from 200 to 500, but the number of VUs doesn't change. We need to either fix the example or fix the title or description... Or, better yet, have examples and explanations for both `ramping-arrival-rate` and `ramping-vus` scenarios, since you can have `duration: '0'` instant jumps in both types of executors, just for different things.

**Comment by LaserPhaser** [comment]
@na-- fixed

File: src/data/markdown/docs/05 Examples/01 Examples/22 instant-load-increase.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: 'Instant load increase'
+excerpt: 'Scripting example on how to instantly increase the number of VUs or iterations and hold them for a period of time'
+draft: 'false'
+---
+
+One of the common usages of the loadtesting tools it's the so-called stepped arrival rate.
+
+In K6 we can achieve it with the following configuration.
```
Comment (na--) [inline src/data/markdown/docs/05 Examples/01 Examples/22 instant-load-increase.md:9]:
```suggestion
In k6 we can achieve it with the following configuration.
```
we generally use lowercase for the k6 brand

**Review by na-- — APPROVED** [review]
LGTM now (besides the very minor nitpick), thanks! :bow:

File: src/data/markdown/docs/05 Examples/01 Examples/22 instant-load-increase.md (hunk 1 above)
Comment (ppcano) [reply src/data/markdown/docs/05 Examples/01 Examples/22 instant-load-increase.md:9]: I can change this.

**Comment by ppcano** [comment]
Thanks for your contribution, @LaserPhaser. Merging!

**Pull request merged**